### DISCUSSION TITLE
Does renovate avoid to create groups of 1 item?

### DISCUSSION THREAD

**Discussion by namelivia** [renovatebot/renovate#31873] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

38.114.0 on CircleCI

### Please tell us more about your question or problem

I have configured some groups like:

```json
    {
      "groupName": "team1 dependencies",
      "groupSlug": "team1",
      "reviewers": [
        "team1_member1",
        "team1_member2",
      ],
      "matchPackageNames": [
        "package1",
        "package2"
      ],
      "matchUpdateTypes": [
        "minor",
        "patch"
      ]
    },
```

When visiting my Dependabot Dashboard and the rate limited updates Im under the impression that for groups like the one above, if there is an update for only `package1` there is an individual PR updating that package and it is not applying the group name or the reviewers. Same for groups that only have one entry on `matchPackageName`.

Does renovate work like this? Can it be configured?
Thanks!

### Logs (if relevant)

<details><summary>Logs</summary>

```

Replace this text with your logs, between the starting and ending triple backticks

```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
No, grouping doesn't work if there's only one dep being updated. 

Maybe you could use labels for such cases.

**Reply by namelivia** [reply]
Thanks for the quick reply
